### PR TITLE
OSD-6853: CatalogSource by digest

### PR DESCRIPTION
**Pull request by 2uasimojo**
[OSD-6853](https://issues.redhat.com/browse/OSD-6853)

### CODE REVIEW

**Comment by 2uasimojo** [comment]
/hold

Depends-On: https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/17943

**Comment by 2uasimojo** [comment]
/hold cancel

Dep is in.

File: hack/olm-registry/olm-artifacts-template.yaml [hunk 1]
```diff
@@ -8,6 +8,8 @@ parameters:
   required: true
 - name: IMAGE_TAG
   required: true
+- name: REPO_DIGEST
```
Comment (rogbas) [inline hack/olm-registry/olm-artifacts-template.yaml:11]: Are the `REGISTRY_IMG`, `CHANNEL` and `IMAGE_TAG` parameters on this template still needed?

Comment (rogbas) [reply hack/olm-registry/olm-artifacts-template.yaml:11]: It seems only `IMAGE_TAG` is still used

Comment (2uasimojo) [reply hack/olm-registry/olm-artifacts-template.yaml:11]: Yes. They're used under the covers to construct and discover the REPO_DIGEST. [Doc](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/docs/app-sre/continuous-delivery-in-app-interface.md#automatically-generated-parameters)

**Pull request merged**